docker/STAR/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:15.04
+MAINTAINER Andrey V Kartashov "porter@porter.st"
+COPY STAR /usr/local/bin/STAR
```

Comment:
Some of the tools a re there, some of the tools can not be there like the latest samtools which is not yet released but latest github revision has important bug fixes